pagefaultgames/pokerogue
# Add .nvmrc for Node Version Manager

#### Changes

- Added `.nvmrc` file with Node.js version `20.13.1`

#### Purpose

- Ensure all contributors use the same Node.js version
- Simplify development environment setup and avoid version-related issues

#### Testing

1. Run `nvm use` to switch to the specified Node.js version
2. Verify the version with `node -v`
